lineaire_algebra_oplossingen.tex

```diff
@@ -33,8 +33,10 @@
 \usepackage{url}
 
 % Pdfpages zorgt ervoor dat we .pdf bestanden kunnen toevoegen.
+\usepackage{grffile}
```

Comment:
Waarvoor is dit nodig? Dat zou tegenwoordig niet meer nodig mogen zijn. Zie https://www.ctan.org/pkg/grffile